components/fxa-client/ios/FxAClient/FxAccountManager.swift

```diff
@@ -9,6 +9,7 @@ public extension Notification.Name {
     static let accountAuthProblems = Notification.Name("accountAuthProblems")
     static let accountAuthenticated = Notification.Name("accountAuthenticated")
     static let accountProfileUpdate = Notification.Name("accountProfileUpdate")
+    static let accountMigrationFailed = Notification.Name("accountMigrationFailed")
```

Comment:
Out of curiosity, what is the app going to do when it receives this notification? Is it something that we could do for ourselves internally here?

Reply:
I added this for Garvan so he can report migration failures to telemetry

Reply:
I added this for Garvan so he can report migration failures to telemetry

Reply:
... and with `accountMigrationFailed` we show a caution symbol badge on the application menu that only disappears when the user taps to re-sign in to sync.